src/components/views/read/cvForm/Education.tsx

```diff
@@ -36,14 +38,27 @@ const Education: FC<Props> = ({
   educationData,
   onCvValuesChange,
 }) => {
+  const { postAppData, patchAppData, deleteAppData, appDataArray } =
+    useAppDataContext();
+  const educationInfoObject = appDataArray.find(
+    (obj) => obj.type === 'educationInfo',
```

Comment:
Good ! It would be nice if you refactored this type to a constant. How about `const EDUCATION_APP_DATA_TYPE = "educationInfo";` ?

Or even better, define an object to group your appData types together:
```ts
const APP_DATA_TYPES = {
  EDUCATION: 'educationInfo',
  ...
};
```